ingestion-core/src/main/java/com/mozilla/telemetry/ingestion/core/Constant.java

```diff
@@ -29,6 +29,7 @@ private Attribute() {
     public static final String GEO_SUBDIVISION1 = "geo_subdivision1";
     public static final String GEO_SUBDIVISION2 = "geo_subdivision2";
     public static final String GEO_DMA_CODE = "geo_dma_code";
+    public static final String IMPROVE_SUGGEST_EXPERIENCE = "improve_suggest_experience_check";
```

Comment:
Per [schema](https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/main/templates/contextual-services/quicksuggest-impression/quicksuggest-impression.1.schema.json#L66), this should be `improve_suggest_experience_checked`.

My fault, there was a typo in the ticket description.